blocks/core/ff_module/ff_module-form-preview/ff_module-form-preview.xsl

```diff
@@ -20,7 +20,13 @@
                 <xsl:with-param name="data" select="ext:node-set($edit)" />
             </xsl:call-template>
             </dt>
-            <dd class="ff_module-form-preview__list__data">
+            <dd>
+                <xsl:attribute name="class">
+                    <xsl:text>ff_module-form-preview__list__data</xsl:text>
+                    <xsl:if test="@data-ff-attr">
+                        <xsl:text> data-ff-</xsl:text><xsl:value-of select="@data-ff-attr"/>
+                    </xsl:if>
+                </xsl:attribute>
```

Comment:
I might be misunderstanding what's going on here, but wouldn't we want to have those as separate attributes rather than as classes? In other words, isn't the HTML we want to end up with something like:

```html
<dd class="ff_module-form-preview__list__data" data-ff-preview-for="some-locator">
```

(Assuming the purpose of the change is to point the summary field to wherever it should grab its value from)